cmd/juju/cloud/add.go

```diff
@@ -137,6 +148,14 @@ func (c *addCloudCommand) runInteractive(ctxt *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
+	pollster.VerifyURLs = func(s string) (ok bool, msg string, err error) {
+		err = c.Ping(provider, s)
```

Comment:
I think we still need to allow not implemented errors to pass through so we can still interactively bootstrap clouds whose provider hasn't implemented `Ping`.